addons/gunbag/functions/fnc_switchWeaponCallback.sqf

```diff
@@ -0,0 +1,75 @@
+/*
+ * Author: Ir0n1E
+ * Get weapon out of gunbag.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ * 1: Target <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, target] call ace_gunbag_fnc_offGunbagCallback
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_unit", "_target"];
+
+private _gunbag = backpackContainer _target;
+
```

Comment:
Does this really need a empty line after each line of script?

Reply:
I just combined the other 2 functions in that module and there are written the same way.